smart_campus/app/templates/app/base.html

```diff
@@ -8,38 +8,65 @@
   <title>{% block title %}Smart Campus{% endblock %}</title>
   {% block stylesheet %}
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/semantic.min.css" %}">
```

Comment:
By not using CDN, it takes a lots more requests to load this page.
I strongly disagree with this.